package.json

```diff
@@ -48,7 +48,7 @@
     "typescript": "^5.0.4"
   },
   "dependencies": {
-    "@vector-im/compound-design-tokens": "https://github.com/vector-im/compound-design-tokens#0.1.0",
+    "@vector-im/compound-design-tokens": "github:element-hq/compound-design-tokens",
```

Comment:
Does removing the version here basically mean whenever we PR into this repo, it will update the storybook with the tokens from `main`?

Reply:
I've added the version back, I was having issues with Yarn saving the wrong version number in yarn.lock when I included it, but I think I've managed to work around that now.

Reply:
FWIW, I wasn't against that change, would be nice to have the storybook updates be a bit more automatic. Maybe it could be a step within the release workflow if that's preferable?